data-science-onramp/data-cleaning/clean_test.py

```diff
@@ -88,27 +128,26 @@ def get_blob_from_path(path):
 
 def test_clean():
     """Tests clean.py by submitting it to a Dataproc cluster"""
-
     # Submit job to Dataproc cluster
     job_client = dataproc.JobControllerClient(
         client_options={"api_endpoint": f"{CLUSTER_REGION}-dataproc.googleapis.com:443"}
     )
-    response = job_client.submit_job_as_operation(
-        project_id=PROJECT, region=CLUSTER_REGION, job=JOB_DETAILS
+    operation = job_client.submit_job_as_operation(
+        project_id=PROJECT_ID, region=CLUSTER_REGION, job=DATAPROC_JOB
     )
 
     # Wait for job to complete
-    result = response.result()
+    result = operation.result()
 
     # Get job output
     output_location = result.driver_output_resource_uri + ".000000000"
     blob = get_blob_from_path(output_location)
     out = blob.download_as_string().decode("utf-8")
 
     # trip duration
-    assert not is_in_table(r"\d*.\d* s", out)
-    assert not is_in_table(r"\d*.\d* min", out)
-    assert not is_in_table(r"\d*.\d* h", out)
+    assert not is_in_table("s", out)
```

Comment:
Unless I'm misunderstanding, it seems like the dirty data would pass this test because an entry "s" is not in the table?

Reply:
It actually is

Reply:
You are correct